src/main/java/com/github/fabienrenaud/jjb/stream/Deserialization.java

```diff
@@ -383,4 +383,177 @@ static User toUser(Value v) {
             return user;
         }
     }
+
+    @Benchmark
+    @Override
+    public Object antons() throws Exception {
+        return antons(sk.antons.json.parse.JsonScanner.instance(JSON_SOURCE().nextString()));
```

Comment:
Note that performance wise, most libs tend to be a faster when manipulating bytes/streams as input. So if your lib supports byte/stream input, I suggest you to use that.

Reply:
not in my case, I stay with string